src/main/java/frc/robot/subsystems/Subsystems.java

```diff
@@ -20,7 +21,15 @@
 public class Subsystems {
   public final SwerveSubsystem drivetrain = new SwerveSubsystem();
   public final Elevator elevator = new Elevator();
-  public final CoralArm coralArm = new CoralArm();
+
+  public static ArmParameters CORALPARAMETERS = (ArmParameters.CoralArm);
+  public final Arm coralArm = new Arm(CORALPARAMETERS);
```

Comment:
We can just pass `ArmParameters.CoralArm` directly.

```suggestion
  public final Arm coralArm = new Arm(ArmParameters.CoralArm);
```